front/src/model/QuestionModel.js

```diff
@@ -0,0 +1,139 @@
+import {EventEmitter} from "events"
+import RestClient from "../rest/RestClient";
+import model from "./UserModel";
+
+
+
+class QuestionModel extends EventEmitter{
+    constructor() {
+        super();
+        this.state = {
+            questions: [{
+                id: 1,
+                title: "title1",
+                text: "text1",
+                author: "John",
+                creation_date_time: 2019,
+                tags: ["first", "tag"]
+            }, {
+                id: 2,
+                title: "title2",
+                text: "text2",
+                author: "Jack",
+                creation_date_time: 2018,
+                tags: ["second", "tag"]
+            }],
+            newQuestion: {
+                title: "",
+                text: "",
+                tags: ""
+            },
+            searchTagOrTitle: "",
+            currentId: 3
+        };
+    }
+
+    loadQuestions(){
+        const client = new RestClient(model.state.newUser.userName, model.state.newUser.password);
```

Comment:
Would be nicer if you either keep a reference to the client (such that you don't create a new one each time), or maybe move the creation in a different method so you don't have this code all over the place :)